src/celengine/astroobj.h

```diff
@@ -21,7 +21,22 @@ class AstroObject
     AstroCatalog::IndexNumber getIndex() const { return m_mainIndexNumber; }
     void setIndex(AstroCatalog::IndexNumber);
 
+//  Auto Indexing stuff
+    static constexpr AstroCatalog::IndexNumber MaxAutoIndex = UINT32_MAX - 1;
+    static AstroCatalog::IndexNumber getNewAutoIndex() { return m_autoIndex--; }
+    static AstroCatalog::IndexNumber getCurrentAutoIndex() { return m_autoIndex; }
```

Comment:
why `getNewAutoIndex` and `getCurrentAutoIndex` return the same value?

Reply:
Hmm..  Indeed. I wanted avoid long names like `getAutoIndexAndUpdate` but apparently used misleading names.  Feel a bit tired and cant find a good ones...

Reply:
i'd prefer long names